### PR TITLE
Correction of the difference between 001 and 002

### PR DESCRIPTION
**Pull request by AIS-RyoMukai**
hello `@Intelligent2013`
i have made some revisions, could you check them and if there are no problems, please merge them into the main branch. thank you.

Just correcting text and replacing images

### Metanorma PR checklist

 - [ ] Breaking changes (list related PRs)
 - [ ] Documentation update required ([create task for this](https://github.com/metanorma/metanorma.org/issues/new))
 - [ ] External dependency introduced ([documentation update need](https://github.com/metanorma/metanorma.org/issues/new))
 - [ ] Gem with native library introduced

### CODE REVIEW

**Comment by ReesePlews** [comment]
its ok @Intelligent2013 i will take care of this.

File: sources/002-v4/images/355.webp.png
Comment (ReesePlews) [inline sources/002-v4/images/355.webp.png:1]: hello @AIS-RyoMukai in the key 凡例 "RoofSurface"is shown but in the image "RooflSurface" is shown.

File: sources/001-v4/images/259.webp.png
Comment (ReesePlews) [inline sources/001-v4/images/259.webp.png:1]: hello @AIS-RyoMukai in the key 凡例 "RoofSurface"is shown but in the image "RooflSurface" is shown.

File: sources/001-v4/images/260.webp.png
Comment (ReesePlews) [inline sources/001-v4/images/260.webp.png:1]: hello @AIS-RyoMukai in the key 凡例 "RoofSurface"is shown but in the image "RooflSurface" is shown.

File: sources/001-v4/images/263.webp.png
Comment (ReesePlews) [inline sources/001-v4/images/263.webp.png:1]: hello @AIS-RyoMukai in the key 凡例 "RoofSurface"is shown but in the image "RooflSurface" is shown.

Comment (AIS-RyoMukai) [reply sources/001-v4/images/263.webp.png:1]: Thank you for pointing it out. I will correct the image when it is reflected in Ver4.1.

**Comment by AIS-RyoMukai** [comment]
@ReesePlews  
The compilation was successful, so I checked the zip.
There are some Japanese errors, images and code definitions that seem to be being corrected, but they will be corrected next time.
Could you please check it and merge it into the main branch if there are no problems?

**Review by ReesePlews — APPROVED** [review]

**Comment by ReesePlews** [comment]
hello @AIS-RyoMukai i have merged the branch, do not delete it, we should  keep it a while if we need to go back.

it is important for you to check the PDF as you did.  please continue to do that for future revisions. if something is wrong, let me know right away and i will ask the dev team.

**Pull request merged**